### ISSUE TITLE
Set up Read the Docs

### ISSUE THREAD

**Issue by ssnn-airr** [airr-community/rep-cred#2]
I have started setting up Read the Docs, but there's is still work to do. I am going to to copy the `airr-standards` style, for consistency across projects.

**Comment by schristley** [comment]
If this is an AIRR-C product, which I assume it is as it's in the airr-community github, I think the documentation should be consolidated in the current docs and not made separate. My suggestion is to put it under the `Software` section.

**Comment by ssnn-airr** [comment]
Makes sense. That would mean saving the rst files in `airr-standards/docs/packages/repecred-R`, right? And update `airr-standards/docs/standards/software.rst`.

**Comment by williamdlees** [comment]
Can we keep the release cycles separate, please? not sure if that's possible in a single doc, or whether we would have to have two documents linked to each other - but I think it's important that new versions can be released independently.

**Comment by schristley** [comment]
> Can we keep the release cycles separate

I see, good point. I don't think we've discussed this much for how to handle an expanding number of software tools within the AIRR-C github. Maybe @javh has some ideas for RTD, setup an umbrella RTD like Immcantation? Maybe rep-cred should have its own RTD, and the airr-standards RTD just has some basic text and links like with ogrdb?

It is also important that maintenance/admin privileges don't get too disjoint, and there's always a sufficient core of people that can manage stuff when needed...

**Comment by javh** [comment]
My inclination would be to have a skeleton page that links out to the dedicated/separate docs page for the package. Linked to "stable" so we don't have to track versions. It's a lot more practical to build the docs from the same repo the code resides in.

So, yeah, like what we did for OGRDB:
https://docs.airr-community.org/en/latest/ogrdb/ogrdb.html

**Comment by ssnn-airr** [comment]
> I see, good point. I don't think we've discussed this much for how to handle an expanding number of software tools within the AIRR-C github.

Also, there are things like code style, code of conduct... that could be general for all packages? Contribution guidelines could be a little different though, as I think this has more to do with how a particular team works in a particular project.

**Comment by ssnn-airr** [comment]
Until we decide something, and I have a working version of the docs, I will use `rep-cred/docs`.
